core-primitives/teerex-storage/src/lib.rs

```diff
@@ -15,7 +15,7 @@ pub trait StoragePrefix {
 
 impl StoragePrefix for TeeRexStorage {
 	fn prefix() -> &'static str {
-		"SubstrateeRegistry"
+		"Teerex"
```

Comment:
Note this is not yet in master:
* https://github.com/integritee-network/integritee-node/pull/51

Reply:
Edit: It is in master now.